docs/content/latest/faq/yugabyte-platform.md

```diff
@@ -14,92 +14,85 @@ isTocNested: false
+For a complete list of prerequisites, refer to [Prerequisites](../../yugabyte-platform/install-yugabyte-platform/prerequisites/).
 
-## What are the OS requirements and permissions to run the YugabyteDB data nodes?
+## What are the OS requirements and permissions to run YugabyteDB data nodes?
 
-Prerequisites for the YugabyteDB data nodes are listed [here](../../../deploy/multi-node-cluster/#prerequisites).
+Prerequisites for YugabyteDB data nodes are listed in the YugabyteDB [Deployment checklist](../../../deploy/checklist).
 
 ## How are the build artifacts packaged and stored for Yugabyte Platform?
 
-The Yugabyte Platform software is packaged as a set of Docker container images hosted on the [Quay.io](https://quay.io/) container registry and managed by the [Replicated](https://www.replicated.com/) management tool. Installation of the admin console starts with installing Replicated on a Linux host. Replicated installs the [docker-engine](https://docs.docker.com/engine/), the Docker container runtime, and then pulls its own container images the Replicated.com container registry. The Yugabyte Platform (YugaWare) then becomes a managed application of Replicated, which starts by pulling the Yugabyte Platform (`yugaware`) container images from Quay.io for the very first time. Replicated ensures that the Yugabyte Platform remains highly available as well as allows for instant upgrades by simply pulling the incremental container images associated with a newer YugaWare release. Note that if the host running the YugabyteDB Admin Console does not have Internet connectivity, then a fully air-gapped installation option is also available.
+Yugabyte Platform software is packaged as a set of Docker container images hosted on the [Quay.io](https://quay.io/) container registry and managed by the [Replicated](https://www.replicated.com/) management tool. Replicated ensures that Yugabyte Platform remains highly available, and allows for instant upgrades by simply pulling the incremental container images associated with a newer platform release. If the host running the Yugabyte Platform console does not have Internet connectivity, a fully air-gapped installation option is also available.
 
-The data node software is packaged into the Yugabyte Platform application. Yugabyte Platform distributes and installs the data node software on the hosts identified to run the data nodes. Since it's already packaged into existing artifacts, the data node does not require any Internet connectivity.
+Installation of the admin console starts with installing Replicated on a Linux host. Replicated installs the [docker-engine](https://docs.docker.com/engine/), the Docker container runtime, and then pulls its own container images from the Replicated.com container registry. Yugabyte Platform then becomes a managed application of Replicated, which starts by pulling the Yugabyte Platform (`yugaware`) container images from Quay.io for the very first time. 
```

Comment:
I mentioned earlier , about linking to the [Replicated.com container registry](https://help.replicated.com/docs/native/getting-started/docker-registries/). Just adding it again.